src/main/java/net/neoforged/neoforge/client/ClientHooks.java

```diff
@@ -435,8 +435,8 @@ public static void onFogRender(FogRenderer.FogMode mode, FogType type, Camera ca
         }
     }
 
-    public static ViewportEvent.ComputeCameraAngles onCameraSetup(GameRenderer renderer, Camera camera, float partial) {
-        ViewportEvent.ComputeCameraAngles event = new ViewportEvent.ComputeCameraAngles(renderer, camera, partial, camera.getYRot(), camera.getXRot(), 0);
+    public static ViewportEvent.ComputeCameraAngles onCameraSetup(Camera camera, float yaw, float pitch, float partial) {
+        ViewportEvent.ComputeCameraAngles event = new ViewportEvent.ComputeCameraAngles(Minecraft.getInstance().gameRenderer, camera, partial, yaw, pitch, 0);
         NeoForge.EVENT_BUS.post(event);
         return event;
     }
```

Comment:
Should we consider inlining this method into the patch? Since what this method does can be replaced with a `var event = NeoForge.EVENT_BUS.post(new ViewportEvent.ComputeCameraAngles(...))` line, and the method is called only once.

Reply:
I'm not a fan of inlining this into the patch because it makes the resulting line extremely long, about 275 characters to be exact, even with `var` instead of writing out the event type for the variable.